### PR TITLE
blog: add v0.17.3.0 release notes

### PR DESCRIPTION
**Pull request by selsta**

### CODE REVIEW

File: _posts/2021-12-04-monero-0.17.3.0-released.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+image: /blog/assets/2020-09-17-monero-0.17-released/oxygen-orion.png
+---
+
+<div align="center">
+   <img src="{{ page.image }}" width="350px">
+ </div>
+
+# Overview
+
+This is the v0.17.3.0 point release of the Monero software. This release contains support for P2Pool.
```
Comment (erciccione) [inline _posts/2021-12-04-monero-0.17.3.0-released.md:16]:
```suggestion
This is the v0.17.3.0 point release of the Monero software. This release contains support for [P2Pool]({{ site.baseurl }}/2021/10/05/p2pool-released.html).
```

Comment (selsta) [reply _posts/2021-12-04-monero-0.17.3.0-released.md]: done

File: _posts/2021-12-04-monero-GUI-0.17.3.0-released.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+<div align="center">
+    <img src="{{ page.image }}" width="350px">
+</div>
+
+# Overview
+
+This is the v0.17.3.0 point release of the Monero GUI software.
+
+[The latest CLI release notes can be found on the precedent blog post]({{ site.baseurl_root }}/2021/12/04/monero-0.17.2.0-released.html)
```
Comment (erciccione) [inline _posts/2021-12-04-monero-GUI-0.17.3.0-released.md:18]:
```suggestion
[The latest CLI release notes can be found on the precedent blog post]({{ site.baseurl_root }}/2021/12/04/monero-0.17.3.0-released.html)
```

File: _posts/2021-12-04-monero-GUI-0.17.3.0-released.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+The complete list of changes is [available on GitHub](https://github.com/monero-project/monero-gui/compare/v0.17.2.3...v0.17.3.0), along with [the source code](https://github.com/monero-project/monero-gui/tree/v0.17.3.0).
+
+# Contributors for this Release
+
+This release was the direct result of 66 people who worked, largely unpaid and altruistically, to put out 93 commits containing 31902 new lines of code. We'd like to thank them very much for their time and effort. In no particular order they are:
+
+- selsta
+- luigi1111
+- weblate
```
Comment (erciccione) [inline _posts/2021-12-04-monero-GUI-0.17.3.0-released.md:41]: This should be removed.

**Review by erciccione — CHANGES_REQUESTED** [review]

File: _posts/2021-12-04-monero-GUI-0.17.3.0-released.md (hunk 2 above)
Comment (selsta) [reply _posts/2021-12-04-monero-GUI-0.17.3.0-released.md]: done

**Review by erciccione — APPROVED** [review]

**Comment by binaryFate** [comment]
Hashes are ok

**Pull request merged**